crates/turbopack/tests/node-file-trace.rs

```diff
@@ -244,6 +245,13 @@ static ALLOC: turbo_tasks_malloc::TurboMalloc = turbo_tasks_malloc::TurboMalloc;
             .expected_stderr("Cannot find module 'sub/@' or its corresponding type declarations")
     ),
 )]
+#[cfg_attr(
+    //[TODO]: WEB-1188 reenable windows once fixed.
+    not(all(feature = "bench_against_node_nft", target_os = "windows")),
```

Comment:
```suggestion
    not(any(feature = "bench_against_node_nft", target_os = "windows")),
```

this should be any, shouldn't it?